backoff/__init__.py

```diff
@@ -0,0 +1,36 @@
+# coding:utf-8
+"""
+Function decoration for backoff and retry
+
+This module provides function decorators which can be used to wrap a
+function such that it will be retried until some condition is met. It
+is meant to be of use when accessing unreliable resources with the
+potential for intermittent failures i.e. network resources and external
+APIs. Somewhat more generally, it may also be of use for dynamically
+polling resources for externally generated content.
+
+For examples and full documentation see the README at
+https://github.com/litl/backoff
+"""
+from backoff._decorator import on_predicate, on_exception
+from backoff._jitter import full_jitter, random_jitter
+from backoff._wait_gen import constant, expo, fibo
+
+__all__ = [
+    'on_predicate',
+    'on_exception',
+    'constant',
+    'expo',
+    'fibo',
+    'full_jitter',
+    'random_jitter'
+]
+
+# make pyflakes happy
```

Comment:
I just tried to switch it, but it broke Python 2.6 support. I know this is old (!) but someone submitted patches for it a while back, and if I were going to drop support I'd need to think about it a bit more first. Also, it may be possible to work around that particular issue in travis, but I'll have to come back to it.  Thanks for the recommendation.